packages/contracts/src/test/Integration.t.sol

```diff
@@ -57,7 +58,8 @@ contract Integration is Test {
         // TODO
         //game.createGame(2, game.allowAnyPlayerAndDeck);
         game.createGame(2);
-        inventory.getDeck(player1, 0);
+        inventory.getDeck(player2, 0); // player1 has card id of 49-72 inclusive
+        inventory.getDeck(player2, 0); // player2 has card id of 73-96 inclusive
```

Comment:
Due to deploy script airdropping the two first anvil addresses. We probably shouldn't rely on this staying like this, but fine in the comments. Maybe good to add `// ids are not starting at 1 because the deploy script currently airdrops some addresses, might change`

Reply:
Done!